pins:
    team_switch: 23
    team_feedback_led: 21
    strategy_switch: 0
    strategy_feedback_led: 20
    initialization_switch: 0
    initialization_feedback_led: 0
    start: 12
